src/Chainweb/Chainweb.hs

```diff
@@ -205,6 +207,7 @@ defaultChainwebConfiguration v = ChainwebConfiguration
     { _configChainwebVersion = v
     , _configNodeId = NodeId 0 -- FIXME
     , _configMiner = defaultEnableConfig defaultMinerConfig
+    , _configCoordinator = False
```

Comment:
`False` by default. Average people running full nodes will likely not want arbitrary `/mining/work/` calls to be allowed.